Gemfile.lock

```diff
@@ -13,6 +13,7 @@ GEM
     eventmachine (1.2.7)
     ffi (1.16.3)
     forwardable-extended (2.6.0)
+    google-protobuf (3.25.3-arm64-darwin)
     google-protobuf (3.25.3-x86_64-darwin)
```

Comment:
I don't know ruby gem deps... Do you pin one version for each architecture?

Reply:
that's actually a great question. I noticed it before and I had been manually removing the gem file from the commits, but it looks like I forgot on the last one. Obviously not a long-term solution. 

TBH, I'm not sure if it even matters if you have multiple different arch specific dependencies, but it's probably better to not.

I believe I need to add the architecture to the bundler [as described in this SO question](https://stackoverflow.com/questions/77738224/how-to-install-arm64-linux-based-ruby-gems-on-a-m1-arm64-darwin-based-mac). Let me double check if that works as expected.